cmd/web/web.go

```diff
@@ -63,11 +77,23 @@ func NewWebCmd() *cobra.Command {
 	})
 
 	webCmd.AddCommand(serveCmd)
+}
 
-	return webCmd
+func addPruneCmd(webCmd *cobra.Command) {
+	var olderThan uint
+
+	pruneCmd := &cobra.Command{
+		Use:   "prune-events",
+		Short: "Prune events older than",
+		Run:   prune,
+	}
+
+	pruneCmd.Flags().UintVar(&olderThan, "older-than", 10, "Prune data discoveryu events older than <value> days.")
+
+	webCmd.AddCommand(pruneCmd)
 }
 
-func serve(cmd *cobra.Command, args []string) {
+func serve(_ *cobra.Command, _ []string) {
```

Comment:
I don't know why golang doesn't complain when the function variables are not used hehe

Reply:
it does complain at linting level, but using the blackhole operator makes the linter happy (and also it is easier to understand that we are not using them)

Reply:
(also to say we should drop a linter in when we have some time :D :D)

Reply:
Yes, it is much better than leaving the not used parameter. It was just an internal thought hehe